### PR TITLE
onboard kaoto

### PR DESCRIPTION
**Pull request by durandom**
fixes https://github.com/operate-first/support/issues/969

### CODE REVIEW

File: cluster-scope/base/core/namespaces/kaoto/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: user.openshift.io/v1
+kind: Namespace
+metadata:
+  annotations:
+    op1st/docs: https://kaoto.io/
+    op1st/onboarding-issue: https://github.com/operate-first/support/issues/969
+    op1st/project-owner: kaoto
+    openshift.io/display-name: Kaoto Integrations
+    openshift.io/requester: kaoto
```
Comment (durandom) [inline cluster-scope/base/core/namespaces/kaoto/namespace.yaml:9]: @mmelko please check if you're happy with the links here

Comment (mmelko) [reply cluster-scope/base/core/namespaces/kaoto/namespace.yaml:9]: I'd change openshift.io/display-name to `Kaoto` too. `Kaoto Integrations` would suggest that Kaoto is actually an integration framework which is not.

**Pull request merged**